docs/standard/commandline/syntax.md

```diff
@@ -0,0 +1,577 @@
+### Short-form aliases
+
+In general, we recommend that you minimize the number of short-form option aliases that you define.
+
+In particular, avoid using any of the following aliases differently than their common usage in the .NET CLI and other .NET command-line apps:
+
+* `-i` for `--interactive`.
+
+  This option signals to the user that they may be prompted for inputs to questions that the command needs answered. For example, prompting for a username.
+
+* `-o` for `--output`.
+
+  Some commands produce files as the result of their execution. This option should be used to help determine where those files should be located. In cases where a single file is created, this option should be a file path. In cases where many files are created, this option should be a directory path.
+
+* `-v` for `--verbosity`.
+
+  Commands often provide output to the user on the console; this option is used to specify the amount of output the user requests. For more information, see [The `--verbosity` option](#the---verbosity-option) later in this article.
+
+There are also some aliases with common usage limited to the .NET CLI. You can use these aliases for other options in your apps, but be aware of the possibility of confusion.
+
+* `-c` for `--configuration`
+
+  This option often refers to a named Build Configuration, like `Debug` or `Release`. You can use any name you want for a configuration, but most tools are expecting one of those. This setting is often used to configure other properties in a way that makes sense for that configuration&mdash;for example, doing less code optimization when building the `Debug` configuration.
+
+* `-f` for `--framework`
+
+  This option is used to select a single [Target Framework Moniker (TFM)](../frameworks.md) to execute for, so if your CLI application has differing behavior based on which TFM is chosen, you should support this flag.
+
+* `-p` for `--property`
+
+  If your application eventually invokes MSBuild, the user is going to need to customize that call in some way. This option allows for MSBuild properties to be provided on the command line and passed on to the underlying MSBuild call. If your app doesn't use MSBuild but needs a set of key-value pairs, consider using this same option name to take advantage of users' expectations.
+
+* `-r` for `--runtime`
+
+  If your application can run on different runtimes, or has runtime-specific logic, consider supporting this option as a way of specifying a [Runtime Identifier](../../core/rid-catalog.md). If your app supports --runtime, consider supporting `--os` and `--arch` also. These options let you specify just the OS or the architecture parts of the RID, leaving the part not specified to be determined from the current platform. For more information, see d[dotnet publish](../../core/tools/dotnet-publish.md).
+
+### Short names
+
+Make command names as short and easy to spell as possible. For example, if `class` is clear enough don't make the command `classification`.
```

Comment:
@jonsequitur For this section, if a name is in Pascal instead of kebab, will we change it?

```suggestion
Make names for commands, options, and arguments as short and easy to spell as possible. For example, if `class` is clear enough don't make the command `classification`.
```